 build/cmake/lib/CMakeLists.txt | 6 +++++-
 1 file changed, 5 insertions(+), 1 deletion(-)

diff --git a/build/cmake/lib/CMakeLists.txt b/build/cmake/lib/CMakeLists.txt
index ec8480d5..612f667f 100644
--- a/build/cmake/lib/CMakeLists.txt
+++ b/build/cmake/lib/CMakeLists.txt
@@ -135,7 +135,7 @@ if (ZSTD_BUILD_STATIC)
             OUTPUT_NAME ${STATIC_LIBRARY_BASE_NAME})
 endif ()
 
-if (UNIX OR MINGW)
+if (TRUE)  # conda-forge patch: do pkg-config on MSVC too, for libtiff
     # pkg-config
     set(PREFIX "${CMAKE_INSTALL_PREFIX}")
     set(EXEC_PREFIX "\${prefix}")
